TransferMethod/Sources/View/Widget/TextWidget.swift

```diff
@@ -92,7 +92,7 @@ class TextWidget: AbstractWidget {
     }
 
     @objc
-    private func textFieldDidChange() {
+    func textFieldDidChange() {
```

Comment:
why is private removed?

Reply:
because we are overriding it in `ExpiryDateWidget`